ipinfo/cmd_config.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/pflag"
+)
+
+func printHelpConfig() {
+	fmt.Printf(
+		`Usage: %s config [<opts>] [<enable | disable>]
```

Comment:
We discussed this and the UI model agreed was to do it like:

```bash
$ ipinfo config cache=enable
$ ipinfo config cache=disable
$ ipinfo config <key>=<value>
```

Reply:
Also, @rm-Umar can you make an issue to look into making it so that the IPinfo token we store currently in `~/.config/ipinfo/token` is also stored inside the new config file? That way it can be even manually configured in the future via:

```bash
$ ipinfo config token=<tok>

# but still allow login to accept token, etc. - it will just put into the config file.
$ ipinfo login
```